packages/frontend/src/components/Auction/BidForm.tsx

```diff
@@ -16,7 +18,9 @@ export const BidForm = ({ minimumBid, bids }: BidProps) => {
   const { account } = useEthers()
   const etherBalance = useEtherBalance(account)
   const [bid, setBid] = useState(minimumBid)
-  const isBadAmount = etherBalance !== undefined && bid.lt(etherBalance)
+  const insufficientAmount = etherBalance !== undefined && bid.lt(etherBalance)
```

Comment:
```suggestion
  const insufficientAmount = !!etherBalance && bid.lt(etherBalance)
```
Terser but hackier looking, I prefer this syntax but it's not much of a difference